nipype/interfaces/robex/preprocess.py

```diff
@@ -0,0 +1,84 @@
+import os
+from pathlib import Path
+
+from nipype.interfaces.base import (
+    TraitedSpec,
+    CommandLineInputSpec,
+    CommandLine,
+    File,
+    traits,
+    isdefined,
+)
+from nipype.utils.filemanip import split_filename
+
+
+class RobexInputSpec(CommandLineInputSpec):
+    in_file = File(
+        desc="Input volume", exists=True, mandatory=True, position=0, argstr="%s"
+    )
+    out_file = File(
+        desc="Output volume",
+        position=1,
+        argstr="%s",
+        hash_files=False,
+        name_template='%s_brain',
+        keep_extension=True,
+    )
+    out_mask = File(
+        desc="Output mask",
+        position=2,
+        argstr="%s",
+        hash_files=False,
+        name_template='%s_brainmask',
+        keep_extension=True,
+    )
+    seed = traits.Int(desc="Seed for random number generator", position=3, argstr="%i")
+
+
+class RobexOutputSpec(TraitedSpec):
+    out_file = File(desc="Output volume", exists=True)
+    out_mask = File(desc="Output mask", exists=True)
```

Comment:
We haven't been entirely consistent before, but `exists=True` is not a great check for output specs.

```suggestion
    out_file = File(desc="Output volume")
    out_mask = File(desc="Output mask")
```